modules/virt-supported-features-for-virtual-storage-matrix.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-supported-features-for-virtual-storage.adoc
+
+[id="virt-supported-features-for-virtual-storage-matrix_{context}"]
+= Supported features for {VirtProductName} virtual storage
+
+.{VirtProductName} virtual storage feature support matrix
+[cols="40%,15%,15%,15%,15%"]
+|===
+||Virtual machine live migration  |Host-assisted virtual machine disk cloning |Storage-assisted virtual machine disk cloning |Virtual machine import
+
+|{product-title} container storage: RBD block volumes
+|Supported
+|Supported
+|Supported
+|Supported
+
+|{product-title} container storage: RBD filesystem volumes
+|Not supported
+|Supported
+|Supported
+|Supported
+
+|{VirtProductName} hostpath provisioner
+|Not supported
+|Supported
+|Not supported
+|Not supported
+
+|Other multi-node writable storage {nbsp} (a)
```

Comment:
v2v doesnt work with RWX

Reply:
We do not work in block mode

Reply:
Wait, what?  Why not?  Is the reason scratch space?  If so, then it's not a problem with ceph because the target pvc can be block mode and the scratch space can be file mode.

Reply:
Again, no change per @aglitke .